src/encoded/schemas/external_quality_metric.json

```diff
@@ -0,0 +1,106 @@
+{
+    "title": "ExternalQualityMetric",
+    "description": "Schema for externally-generated quality control data for a file.",
+    "$id": "/profiles/external_quality_metric.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": [
+        "submission_centers",
+        "submitted_id",
+        "qc_values"
+    ],
+    "identifyingProperties": [
+        "submitted_id",
+        "accession",
+        "aliases",
+        "uuid"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/category"
+        },
+        {
+            "$ref": "mixins.json#/description"
+        },
+        {
+            "$ref": "mixins.json#/modified"
+        },
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/status"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/submitted_id"
+        },
+        {
+            "$ref": "mixins.json#/tags"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        }
+    ],
+    "properties": {
+        "accession": {
+            "accessionType": "EQ"
+        },
+        "schema_version": {
+            "default": "1"
+        },
+        "submitted_id": {
+            "pattern": "^[A-Z0-9]{3,}_EXTERNAL-QUALITY-METRIC_[A-Z0-9-_.]{4,}$"
+        },
+        "category": {
+           "items": {
+                "type": "string",
+                "suggested_enum": [
+                    "DuplexSeq"
+                ]
+            }
+        },
+        "qc_values": {
```

Comment:
Can you not merge this entire field from quality metric as is without the subfields? If a description is missing it can probably just go on quality metric.

Reply:
I did it this way because the other properties within `qc_values`, `visible` and `flag` seemed to be things that were specific to internal QualityMetric items. Would you recommend merging the whole property and setting those to `restricted_fields` or something else?